source/endpoints/plugins/business_logic/RequiredInfo.py

```diff
@@ -0,0 +1,18 @@
+class BusinessLogicRequiredInfo:
```

Comment:
To be honest I do not really see the benefits of having a wrapper around the plugin manager as almost all the this is does is calling the plugin_manager.